packages/dicomImageLoader/src/shared/decoders/decodeLittleEndian.ts

```diff
@@ -32,7 +32,11 @@ async function decodeLittleEndian(
       offset = 0;
     }
 
-    imageFrame.pixelData = new Float32Array(arrayBuffer, offset, length / 4);
+    if (imageFrame.pixelRepresentation === 0) {
```

Comment:
Shouldn't this check if imageFrame.floatPixelData or doubleFloatPixel data exist?  It should use Float32Array only for floatPixelData